### PR TITLE
Quota & negative inflights

### PR DESCRIPTION
**Pull request by ghost**
- Improve tests to ensure S3 will behave correctly if we use negative inflights
- Add support for quotas in backbeat routes (we considered only S3 router till now).

### CODE REVIEW

File: lib/routes/routeBackbeat.js
Comment (ghost) [inline lib/routes/routeBackbeat.js:1]: Backbeat routes will be tested in Zenko CI - existing tests are not up to date, plus they are skipped.

Comment (francoisferrand) [reply lib/routes/routeBackbeat.js:1]: this was a separate ticket (cldsrv-586) :/

Comment (ghost) [reply lib/routes/routeBackbeat.js:1]: Please feel free to do it separately, I will drop the commit

**Review by francoisferrand — APPROVED** [review]

**Review by KillianG — APPROVED** [review]

**Pull request merged**